### PR TITLE
feat: added dark mode to login, logout, password reset

### LINKED ISSUE #11763: [CAL-2596] dark mode for login page
needed for "calendar overlay":

https://github.com/calcom/cal.com/assets/8019099/13563c18-1186-4bc4-b9bc-ab865551d469

<sub>[CAL-2596](https://linear.app/calcom/issue/CAL-2596/dark-mode-for-login-page)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
CleanShot 2023-10-09 at 10 36 09@2x

CleanShot 2023-10-09 at 10 35 29@2x

### CODE REVIEW

File: apps/web/pages/auth/forgot-password/[id].tsx [hunk 1]
```diff
@@ -141,7 +141,6 @@ export default function Page({ requestId, isRequestExpired, csrfToken }: Props)
   );
 }
 
-Page.isThemeSupported = false;
```
Comment (PeerRich) [inline apps/web/pages/auth/forgot-password/[id].tsx:144]: dont force light mode

File: apps/web/pages/auth/forgot-password/index.tsx [hunk 2]
```diff
@@ -126,8 +126,9 @@ export default function ForgotPassword({ csrfToken }: { csrfToken: string }) {
             />
             <div className="space-y-2">
               <Button
-                className="w-full justify-center"
```
Comment (PeerRich) [inline apps/web/pages/auth/forgot-password/index.tsx:129]: somehow our primary button does not have dark mode??

File: apps/web/pages/auth/login.tsx [hunk 3]
```diff
@@ -225,7 +225,7 @@ export default function Login({
                 type="submit"
                 color="primary"
                 disabled={formState.isSubmitting}
-                className="w-full justify-center">
```
Comment (PeerRich) [inline apps/web/pages/auth/login.tsx:228]: same

**Review by sean-brydon — APPROVED** [review]
Tested all pages - LGTM 

Primary issue you noticed is due to us forcing CSS vars as thats how we handle brand colour - this will have to do for now

**Pull request merged**